### PR TITLE
REGAPIC and Compute support

### PR DESCRIPTION
**Pull request by jskeet**
This adds support for "regapic" as both a generator and package type, and then moves the Compute.V1 API from the regapic branch of gax-dotnet.

### CODE REVIEW

**Comment by jskeet** [comment]
cc @viacheslav-rostovtsev

File: apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    public sealed class AcceptanceTest
+    {
+        private readonly ITestOutputHelper _output;
+        private readonly ComputeFixture _fixture;
+
+        public AcceptanceTest(ComputeFixture fixture, ITestOutputHelper output) =>
+            (_fixture, _output) = (fixture, output);
+
+        [Fact]
+        public void CreateDeleteIPAddress()
```
Comment (amanda-tarafa) [inline apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs:36]: I would write individual tests with proper resuource initialization and cleanup after.
We wouldn't need to write so much to the output then.
Basically, this is a big departure from how we write integration tests in this repo and I would prefrer if we stick to one way of writing tests. I think these were going to be worked on more, so maybe something to take into account.

File: apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            {
+                _output.WriteLine($"Deleting address with the name: {addressName}");
+                Operation deleteOp = addressesClient.Delete(projectId, region, addressName);
+                _output.WriteLine($"Operation to delete address: {deleteOp.Name} status {deleteOp.Status}; start time {deleteOp.StartTime}");
+                deleteOp = PollForCompletion(deleteOp, "delete");
+                _output.WriteLine($"Operation to delete address completed: status {deleteOp.Status}; start time {deleteOp.StartTime}; end time {deleteOp.EndTime}");
+            }
+        }
+
+        private Operation PollForCompletion(Operation operation, string alias)
```
Comment (amanda-tarafa) [inline apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs:90]: Should this be a public extension method in the library, or is this something that will be generated better later?

**Review by amanda-tarafa — APPROVED** [review]
LGTM

File: apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs (hunk 2 above)
Comment (jskeet) [reply apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs:90]: The latter, I believe. At least, that's what I'm hoping.

File: apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs:36]: We've got a *few* tests that are like this - particularly in Storage, for one access control feature. (Can't remember the name now.)

I did consider doing resource clean up in the disposal part - listing all IP addresses either specifying a filter (if there's such a thing as a "name starts with" filter) or deleting everything starting with "testaddr". Or possibly generating a per-test prefix. Thoughts on that?

Comment (amanda-tarafa) [reply apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs:36]: You'd probably need a per-test prefix, there's a chance that Windows and Ubuntu are running at the same time and one tramples on the other's IPs.

Comment (jskeet) [reply apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs:36]: Yup - it'll be part of the test fixture.

**Comment by jskeet** [comment]
Merging, and will then create a follow-up PR for cleanup.

**Pull request merged**